sdk/eventhub/event-processor-host/test/eph.spec.ts

```diff
@@ -45,11 +48,12 @@ describe("EPH", function(): void {
         }
       );
       const context = host["_context"];
-      const ua = "/js-event-processor-host=1.0.5";
-      context.userAgent.should.equal(ua);
+      const uaPrefix = "azsdk-js-azureeventprocessorhost/1.0.5 ";
```

Comment:
Any idea why this is 1.0.5? I wonder if that is the version that was hard-coded before